lib/daru/vector.rb

```diff
@@ -1548,5 +1576,13 @@ def resort_index vector_index, opts
       end
         .tap { |res| res.reverse! unless opts[:ascending] }
     end
+
+    def table data
+      Formatters::Table.format(
+        data,
+        headers: ['']*3,
+        row_headers: ['']*frequencies.size
```

Comment:
Well I tested it but it is not showing the data without the headers. There is no such no header option. Without headers only an empty string is displayed. Also there are no tests to justify the same. Should I fix this in this PR itself?